scylla/src/transport/session_test.rs

```diff
@@ -1978,3 +1978,47 @@ async fn test_unprepared_reprepare_in_caching_session_execute() {
     all_rows.sort();
     assert_eq!(all_rows, vec![(1, 2, 3), (1, 3, 2)]);
 }
+
+#[tokio::test]
+async fn test_views_in_schema_info() {
+    let _ = tracing_subscriber::fmt::try_init();
+
+    let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+    let session = SessionBuilder::new().known_node(uri).build().await.unwrap();
+    let ks = unique_name();
+
+    session.query(format!("CREATE KEYSPACE IF NOT EXISTS {} WITH REPLICATION = {{'class' : 'SimpleStrategy', 'replication_factor' : 1}}", ks), &[]).await.unwrap();
+    session.use_keyspace(ks.clone(), false).await.unwrap();
+
+    session
+        .query("CREATE TABLE t(id int PRIMARY KEY, v int)", &[])
+        .await
+        .unwrap();
+
+    let result = session.query("CREATE MATERIALIZED VIEW mv1 AS SELECT * FROM t WHERE v IS NOT NULL PRIMARY KEY (v, id)", &[]).await;
+    if result.is_err() {
+        // Looks like we're testing on Cassandra without materialized views enabled, skip
```

Comment:
This looks dangerous. What if syntax for creating materialized views changes slightly/there is some other unrelated issue and this test silently stops running at all.

I think if we ever end up testing the driver against Cassandra with materialized views disabled we can just pass a flag to skip it.
https://github.com/scylladb/scylla-rust-driver/blob/6577a07d0e369a8923f4c8e56316ab1b1fdab316/.github/workflows/cassandra.yml#L30